logging/src/logging.cc

```diff
@@ -81,15 +105,56 @@ namespace leatherman { namespace logging {
         dst.imbue(leatherman::locale::get_locale(locale));
 #endif
 
-        boost::log::add_common_attributes();
-
-        // Default to the warning level
-        set_level(log_level::warning);
+        setup_common_logging_config<sink_t>(sink);
 
         // Set whether or not to use colorization depending if the destination is a tty
         g_colorize = color_supported(dst);
     }
 
+    void setup_logging(std::string file_name,
+                       std::ios_base::openmode open_mode,
+                       std::string target,
+                       int rotation_size,
+                       int max_size,
+                       int min_free_space,
+                       std::string locale)
+    {
+        // Use the text file as logging backend, in order to set file rotation
```

Comment:
Although if you're opening a file for logging, imbuing the filesystem makes a bit more sense than forcing it automagically in non-file logging setup...